tools/frrcommon.sh.in

```diff
@@ -164,7 +168,7 @@ daemon_start() {
 
 	daemon_inst "$1"
 
-	ulimit -n $MAX_FDS > /dev/null 2> /dev/null
+	ulimit -n "$MAX_FDS" > /dev/null 2> /dev/null
```

Comment:
This causes an issue if the MAX_FDS is not set (by default it's commented out and not set in `daemons`)

The expression `ulimit -n "" > /dev/null 2> /dev/null` will in this case cause the fail of the FRR daemon startup.
(This is the reason for all the failures in the CI system)

Suggested to write it like this: `[ "$MAX_FDS" != "" ] && ulimit -n "$MAX_FDS" > /dev/null 2> /dev/null` instead

Reply:
fixed

Reply:
fiuxed